scalafix-tests/input/src/main/scala/test/NoInferConfig.scala

```diff
@@ -0,0 +1,17 @@
+/*
+rule = NoInfer
+NoInfer.symbols = [
+  "scala.Predef.any2stringadd"
+  "test.NoInferConfig.B."
+]
+*/
+package test
+
+case object NoInferConfig {
+  case class B()
+  List(B.apply) // assert: NoInfer.b
+  List[B](B()) // assert: NoInfer.apply
```

Comment:
I'm not 100% sure this is what we want. Please see here

https://gitter.im/scalacenter/scalafix?at=59eb60f58808bed73d0ed892

Reply:
This is unexpected behavior IMO. I think we are actually safe to remove this case here https://github.com/scalacenter/scalafix/blob/ff5ebd3833cd3e4b349020f0258aeea67c5f5552/scalafix-core/shared/src/main/scala/scalafix/internal/util/SymbolOps.scala#L57-L59 and that should fix this problem. `A.apply` should not get normalized to `A`